build.sbt

```diff
@@ -31,8 +31,6 @@ lazy val microservice = Project(appName, file("."))
     libraryDependencies ++= AppDependencies.all,
     // To resolve dependency clash between flexmark v0.64.4+ and play-language to run accessibility tests, remove when versions align
     dependencyOverrides += "com.ibm.icu" % "icu4j" % "69.1",
```

Comment:
I tried removing this line to run the accessibility tests, and everything seemed to pass OK?

Reply:
Ah, yes, I'd been meaning to revisit this and forgot. Will remove and resubmit.